### PR TITLE
Add 1230 message support

### PR DESCRIPTION
**Pull request by anth-cole**
Add c decoding support for the 1230 bias message

@scarcanague

### CODE REVIEW

File: c/src/rtcm3_decode.c [hunk 1]
```diff
@@ -685,3 +685,49 @@ int8_t rtcm3_decode_1012(const uint8_t *buff, rtcm_obs_message *msg_1012)
...
+  msg_1230->stn_id = getbitu(buff, bit, 12);
+  bit += 12;
+  msg_1230->bias_indicator = getbitu(buff, bit, 1);
+  bit += 1;
+  /* 3 Reserved bits */
+  bit+= 3;
+  msg_1230->fdma_signal_mask = getbitu(buff, bit, 4);
+  bit += 4;
+  if(msg_1230->fdma_signal_mask & 0x08) {
+    msg_1230->L1_CA_cpb = getbitu(buff, bit, 16) * 0.02;
```
Comment (scarcanague) [inline c/src/rtcm3_decode.c]: Im sure it's right but where do you see in the standard that you have to multiply by 0.02?

File: c/test/rtcm_decoder_tests.c [hunk 2]
```diff
@@ -885,3 +909,51 @@ bool msg1008_equals(const rtcm_msg_1008 *lhs, const rtcm_msg_1008 *rhs)
...
+    return false;
+  }
+
+  if (lhs->fdma_signal_mask != rhs->fdma_signal_mask) {
+    printf("1230 FDMA signal mask not equal %u %u\n",lhs->fdma_signal_mask,rhs->fdma_signal_mask);
+    return false;
+  }
+
+  if (lhs->fdma_signal_mask & 0x08) {
+    if( lhs->L1_CA_cpb - rhs->L1_CA_cpb > 0.01 ) {
```
Comment (scarcanague) [inline c/test/rtcm_decoder_tests.c]: Absolute value of the difference instead?

File: c/src/rtcm3_decode.c (hunk 1 above)
Comment (anth-cole) [inline c/src/rtcm3_decode.c]: These are fields DF423-DF426 in the RTCM standard and they are transmitted with a resolution of 0.02m, so multiplying by 0.02 should bring the units back to m.

File: c/test/rtcm_decoder_tests.c (hunk 2 above)
Comment (anth-cole) [inline c/test/rtcm_decoder_tests.c]: good point, and then I realised I was encoding/decoding unsigned and not signed ints, so good catch!

**Pull request merged**